emails/templates/emails/first_time_user.html

```diff
@@ -0,0 +1,411 @@
+{% comment %}
+  Note that Django only loads strings from `app.ftl`, `brands.ftl`, and `phones.ftl`
+  so make sure all email strings are included in those files.
+{% endcomment %}
+{% load ftl %}
+{% load email_extras %}
+{% withftl bundle='privaterelay.ftl_bundles.main' language=language %}
+
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD XHTML 1.0 Transitional //EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml" lang="{{ request.LANGUAGE_CODE }}" dir="ltr"></html>
+
+<head>
+  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta name="x-apple-disable-message-reformatting">  
+  <title>Firefox Relay</title>
```

Comment:
I think it is worth changing in both places for consistency. I think it would only show up at the preview at`/emails/first_time_user_test`, so not critical.

Reply:
Do we have an idea of what we would want to change it to? Do we want this to be a fluent string?

Reply:
Yeah, it doesn't really matter what you do, but if you forced me at gunpoint to make a call, I'd just remove it in both, just to make sure we only concern ourselves with what the user actually sees.

Reply:
I removed `<title>` from all emails.